mlflow/exceptions.py

```diff
@@ -50,6 +50,10 @@ def __init__(self, message, error_code=INTERNAL_ERROR, **kwargs):
             self.error_code = ErrorCode.Name(error_code)
         except (ValueError, TypeError):
             self.error_code = ErrorCode.Name(INTERNAL_ERROR)
+        if isinstance(message, BaseException):
+            message = f"{message.__module__}.{message.__class__}: {str(message)}"
```

Comment:
If we do this, I'd define `from_exception` class method.

Reply:
Some tests failed in CI but passed in my local machine. I added this for debugging for now.